src/Umbraco.PublishedCache.NuCache/PublishedSnapshotService.cs

```diff
@@ -84,13 +83,21 @@ internal class PublishedSnapshotService : PublishedSnapshotServiceBase
 
         //private static int _singletonCheck;
 
-        public PublishedSnapshotService(PublishedSnapshotServiceOptions options, IMainDom mainDom, IRuntimeState runtime,
-            ServiceContext serviceContext, IPublishedContentTypeFactory publishedContentTypeFactory,
-            IPublishedSnapshotAccessor publishedSnapshotAccessor, IVariationContextAccessor variationContextAccessor,
+        public PublishedSnapshotService(
+            PublishedSnapshotServiceOptions options,
```

Comment:
If ever there was a sign that a class needs redesigned. This is it. 24 parameters!!!

Reply:
I think everyone agrees :P

Reply:
![c4jt321](https://user-images.githubusercontent.com/2056399/100219989-a2a98b80-2f0e-11eb-833c-f3df4491bf9b.png)